docs/src/documentation/05-development/04-migration-guides/01-v16.mdx

```diff
@@ -12,13 +12,49 @@ With this guide, we aim to walk through all the breaking changes and how they ca
 
 This version addresses especially a change in the design tokens, and tailwind classes.
 
-## Token and tailwind classes
+## Breaking changes
+
+### Heading component
+
+A new heading token was added: `title0`. This token is similar to the previously existing `title1` token. Therefore, `title1` token was updated to have new font size and line height values.
+
+With that, the `Heading` component now accepts a new value for the `type` prop: `"title0"`.
+
+Because of that, all occurrences of `Heading` with `type="title1"` must be replaced with `type="title0"`, to avoid the visual change.
```

Comment:
WDYT about making **must be replaced** in bold and include "visual breaking change" in the end?

Reply:
Agreed. Done